api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/WeightMappings.kt

```diff
@@ -55,154 +55,102 @@ internal object WeightMappings {
         }
     }
 
-    private fun getConv2DVariables(layer: Conv2D): Map<String, Pair<String, LongArray>> {
-        val variables = mutableMapOf(
-            Pair("kernel:0", Pair(convKernelVarName(layer.name, dim = 2), layer.kernelShapeArray!!))
-        )
-        if (layer.useBias) {
-            variables["bias:0"] = Pair(convBiasVarName(layer.name, dim = 2), layer.biasShapeArray!!)
-        }
-        return variables
+    private fun getConv2DVariables(layer: Conv2D): Map<String, KVariable> {
+        return mapOfNotNull("kernel:0" to layer.kernel, "bias:0" to layer.bias)
```

Comment:
does it correctly handle the situation when the layer.useBias = false?

Reply:
Yes. If `layer.useBias` is `false`, then `layer.bias` is `null`, which gets filtered out by `mapOfNotNull`.

Reply:
Yes, no I see, it's hidden in this procedure